<launch>
  <node name="exbot_rplidar" pkg="rplidar" type="rplidar">
    <param name="com_path" type="string" value="/dev/ttyUSB0" />
    <param name="com_baudrate" type="int" value="115200" />
    <param name="frame_id" type="string" value="laser_frame" />
    <param name="range_min" type="double" value="0.0" />
    <param name="range_max" type="double" value="13.0" />
  </node>
</launch>